openlibrary/plugins/importapi/import_validator.py

```diff
@@ -15,21 +15,21 @@ class Author(BaseModel):
     name: NonEmptyStr
 
 
-class CompleteBookPlus(BaseModel):
+class CompleteBook(BaseModel):
     """
-    The model for a complete book, plus source_records and publishers.
+    The model for a complete book, plus source_records.
 
-    A complete book has title, authors, and publish_date. See #9440.
+    A complete book has title, authors, and publish_date, as well as
+    source_records. See #9440.
     """
 
     title: NonEmptyStr
     source_records: NonEmptyList[NonEmptyStr]
     authors: NonEmptyList[Author]
-    publishers: NonEmptyList[NonEmptyStr]
```

Comment:
I think to be properly complete and differentiable an edition import _needs_ a publisher.

Reply:
I've undone this change and that will be reflected in the next commit I push.